app/view_objects/assessor_interface/application_forms_show_view_object.rb

```diff
@@ -90,6 +91,18 @@ def management_tasks
            to: :assessment
   delegate :canonical_email, to: :teacher
 
+  def management_task_list_section
+    return nil if management_tasks.blank?
+
+    {
+      title: "Management",
+      items:
+        management_tasks.map do |task|
```

Comment:
Since `management_tasks` is now only being used in one place (here), we can change that method to return the items directly (an array of hashes with a `name` and a `link`) which removes the need to transform it here with `map`.